src/overlay_01_021F1348.c

```diff
@@ -0,0 +1,3 @@
+#include "overlay_01_021F1348.h"
+
```

Comment:
yup this one's still being worked on, but still a large file

Reply:
on closer review this is a file I was working about to work on but realized I needed to decomp some rendering stuff before I tackled it, will remove it since it'll be awhile before I actually get to it